pkg/engine/templates_generated.go

```diff
@@ -37045,6 +36995,22 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
     {{WrapAsVariable "environmentJSON"}}
 {{end}}
 
+disk_setup:
+  /dev/disk/azure/scsi1/lun0:
+    table_type: gpt
+    layout: true
+    overwrite: false
+fs_setup:
+  - label: etcd_disk
+    filesystem: ext4
+    device: /dev/disk/azure/scsi1/lun0
+    extra_opts:
+      - -F
+      - -E
```

Comment:
Why do we need -F and -E?

Reply:
I'm not entirely sure but that's what we have right now in the script https://github.com/Azure/aks-engine/pull/3273/files#diff-68a375c3d9c31b5f27994806a7f564cfL14

Reply:
You can't specify the lazy_itable_init and lazy_journal_init without -E. I think those two options are recommended if formatting disk is a bottleneck during bootstrapping (ext4 formatting is expensive). What it does is that it will delay initializing the inode table and journal node. I think lazy_itable_init is probably not meaningful because it will require the kernel to initialize it at mounting (which seems to be what you're doing immediately after formatting)
As to -F I am not sure of the historical reason for AKS to use it so I can't comment on it